pylintrc

```diff
@@ -11,7 +11,7 @@ ignore=CVS
 
 # Add files or directories matching the regex patterns to the blacklist. The
```

Comment:
should we maybe relax a little on e..g. line lengths? here and for yapf? sometimes it's just simpler to allow slightly longer lines and not have as much of trouble with the wrapped lines which can become annoying (e.g. for imports, comments, deeply nested loops)

Reply:
If I recall properly, though, the values we are using are already a bit more relaxed than the default values for pylint

Reply:
Ah, didn't read the whole pylintrc. We're using 100 chars, which is ok (the default is ~80 which is a little narrow often enough and going way larger leads to trouble when having 2 files next to each other e.g. in a diff). And indeed if I parse the regexp correctly we're allowing comments and web-addresses to go beyond the line limit (and could add other stuff to there if we figure out there's issues).

Is yapf using the same setup that the linter uses? Or will that produce the default settings or has a seperate config? Iirc there was some way to unify all of this, but I don't remember right now how it worked...

Reply:
I guess yapf is using google coding standards with their defaults (as I have not created a config file for that)

Reply:
ok, let's just keep it like that for the moment.